contributingGuides/HYBRID_APP.md

```diff
@@ -0,0 +1,212 @@
+# What is HybridApp?
+Every React Native mobile application needs native codebase with an entry point, known as `ReactRootView`. Depending on the platform the implementation differs. Nevertheless as a rule of thumb we can say that React Native just needs a **native** screen where JavaScript runtime can manipulate components and views.
+
+It means that whenever we create a React Native application from scratch we instantiate it on a (very basic, bare) native codebase. However, we can always use a different, already existing codebase to run React Native. This is how we've built HybridApp - we've created a new `ReactRootView`, and taken all JavaScript code to run it in so-called OldDot.
```

Comment:
> and taken all JavaScript code to run it in so-called OldDot.

and taken all JavaScript code to run it within the existing mobile application (OldDot).

Reply:
Lets just clarify this, as it is the first usage of 'OldDot'